lib/datasource/docker/index.ts

```diff
@@ -631,7 +694,13 @@ export async function getReleases({
     lookupName,
     registryUrl
   );
-  const tags = await getTags(registry, repository);
+  const isQuay = registry.endsWith('quay.io');
```

Comment:
```suggestion
  const isQuay = registry == 'https://quay.io';
```
Should be an exact match. Add a const to `./quay.ts` to reuse `https://quay.io` value